src/components/questions-accordion/QuestionsAccordion.tsx

```diff
@@ -0,0 +1,68 @@
+import { FC, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+
+import ArrowForwardIosSharpIcon from '@mui/icons-material/ArrowForwardIosSharp'
+
+import { AccordionItem, VariantEnum } from '~/types'
+import { styles } from '~/components/questions-accordion/QuestionsAccordion.styles'
+import Accordions from '../accordion/Accordions'
+
+interface QuestionsAccordion {
+  items: AccordionItem[]
+  title?: string
+}
+
+const QuestionsAccordion: FC<QuestionsAccordion> = ({ items, title = '' }) => {

```

Comment:
bad name for component, it can be not only questions

Reply:
maybe something like MultipleAccordionWithTItle?

Reply:
yes something like this, use gpt it generate good names